README.md

```diff
@@ -1,5 +1,36 @@
 # SciPy Proceedings Builder
 
+## Quickstart: Docker Container
+
+If you already have docker engine setup, you should be able to run a version of the proceedings server by running 
+
+```bash
+docker run -it -p 7001:7001 scipyproc/procbuild
+```
+
+Then, if you go to localhost:7001, you should see the server interface, and you
+can manually trigger builds by clicking the button on the right hand side.
+
+This will occupy the terminal, when you want to shut down the server and stop
+running the image you will need to type <kbd>ctrl</kbd><kbd>C</kbd>. This should
+shutdown the server and stop running the image (if you used the above command). 
```

Comment:
This all looks great, thanks @mpacer.

What does "if you used the above command" mean in this context?  Presumably, that's the only command the user is aware of at this point, so it's unclear what alternative command they would have executed that would have not allowed shutting down the server.

Reply:
force pushing to fix it right now